decidim-admin/app/helpers/decidim/admin/officializations_helper.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    module OfficializationsHelper
+      def profile_path(user)
+        Decidim::Core::Engine.routes.url_helpers.profile_path(user&.nickname)
```

Comment:
Does `decidim.profile_path(user&.nickname)` work in this context?

Reply:
Yes. Thanks a lot.